### PR TITLE
[nightwatch] Fix browser.element(s) and element().findElements().

### PR DESCRIPTION
**Pull request by garg3133**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/nightwatch/test/testActions.ts [hunk 1]
```diff
@@ -1,3 +1,7 @@
+import { NightwatchAPI } from "nightwatch";
+
+function isNightwatchAPI(v: NightwatchAPI) {}
```
Comment (vaibhavsingh97) [inline types/nightwatch/test/testActions.ts:3]: Can we move this to a common `util` file?

File: types/nightwatch/test/testActions.ts [hunk 2]
```diff
@@ -46,7 +50,15 @@ it('actions.dragAndDrop', () => {
     browser.perform(async () => {
         const sampleElement = element('.element-class');
         const webElement = await sampleElement.getWebElement();
-        return browser.actions().dragAndDrop(webElement, {x: 12, y: 234});
+        browser.actions().dragAndDrop(webElement, {x: 12, y: 234});
+
+        const webElement2 = sampleElement.findElement();
+        isNightwatchAPI(webElement2);
+        browser.actions().dragAndDrop(await webElement2, {x: 12, y: 234});
```
Comment (vaibhavsingh97) [inline types/nightwatch/test/testActions.ts:57]: Should we also add an example where `webElement2` is passed directly in the actions API?

File: types/nightwatch/index.d.ts [hunk 3]
```diff
@@ -1865,7 +1865,7 @@ export class Element {
     message: string;
     timeout: number;
     getId: () => string;
-    findElement: ElementCommands['findElement'];
+    findElement: ElementCommands['findElement'] & {(): Awaitable<NightwatchAPI, WebElement>};
```
Comment (vaibhavsingh97) [inline types/nightwatch/index.d.ts:1868]: Does this also need to be fixed for `findElements`?

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]

File: types/nightwatch/index.d.ts (hunk 3 above)
Comment (garg3133) [reply types/nightwatch/index.d.ts:1868]: No, this behaviour is specific to `findElement`.

File: types/nightwatch/test/testActions.ts (hunk 1 above)
Comment (garg3133) [reply types/nightwatch/test/testActions.ts]: Move all the common functions to `test/utils.ts`.

**Review by vaibhavsingh97 — APPROVED** [review]

**Comment by vaibhavsingh97** [comment]
Ready to merge

**Pull request merged**